lib/beaker/host/unix/exec.rb

```diff
@@ -154,7 +154,7 @@ def mkdir_p dir
   # Recursively remove the path provided
   # @param [String] path The path to remove
   def rm_rf path
-    execute("rm -rf #{path}")
+    execute("rm -rf \"#{path}\"")
```

Comment:
I recall that this can break things. For example, you can no longer use `~` for a home directory.
```console
$ ls "~/bla"
ls: cannot access '~/bla': No such file or directory
```

That means that it's an API change, not a simple bugfix.

Reply:
Great catch, I didn't think of it. Escaping spaces, addresses the issue you mentioned.